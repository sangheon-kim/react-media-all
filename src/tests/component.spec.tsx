import * as React from "react";
import * as renderer from "react-test-renderer";
import { ChangeMedia } from "..";

test("component testing", () => {
  const widthObj = {
    pc: 1025,
    tb: 768,
    ph: 767,
  };
  const component = renderer.create(
    <ChangeMedia widthObj={widthObj}>test</ChangeMedia>
  );
  const testInstance = component.root;

  expect(testInstance.findByType(ChangeMedia).props.widthObj.pc).toBe(1025);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
