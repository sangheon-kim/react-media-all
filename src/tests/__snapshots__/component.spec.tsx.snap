// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component testing 1`] = `"test"`;
